bl/evaluation/evaluator_test.go

```diff
@@ -144,7 +144,7 @@ type evaluateTestCase struct {
 }
 
 func request_evaluation_all_valid() *evaluateTestCase {
-	validFilePath := "../../internal/fixtures/kube/pass-all.yaml"
+	const validFilePath = "../../fixtures/kube/pass-all.yaml"
```

Comment:
Following our conversation, move this to the top(for instance, line:12) and use it in each test case. This way we won't need to repeat `../../fixtures/kube` every time.

Reply:
Thank you @noaabarki for suggesting me better way and helping me out. Now, it's done.